scripts/fixate-requirements.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import os
+import os.path
+import sys
+from distutils.version import StrictVersion
+
+
+cwd = os.path.abspath(os.getcwd())
+FIXED_REQF = os.path.join(cwd, '../fixed-requirements.txt')
+LOCAL_REQF = os.path.join(cwd, 'requirements.txt')
+GET_PIP = "    curl https://bootstrap.pypa.io/get-pip.py | python"
+
+try:
+    import pip
+    from pip.req import parse_requirements
+except ImportError:
+    print "Download pip:\n", GET_PIP
+    sys.exit(1)
+
+
+def check_pip_version():
+    if StrictVersion(pip.__version__) < StrictVersion('7.0.0'):
```

Comment:
I believe it just needs to be `> 6.0.0`, right? Since `session` argument was added to `parse_requirements` method in 6.0.